src/genie/libs/parser/aos/show_vlan.py

```diff
@@ -0,0 +1,108 @@
+"""show_vlan.py
+
+"""
+import re
+import logging
+
+from genie.metaparser import MetaParser
+from genie.libs.parser.utils.common import Common
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+logger = logging.getLogger(__name__)
+
+
+def regexp(expression):
+    def match(value):
+        if re.match(expression,value):
+            return value
+        else:
+            raise TypeError("Value '%s' doesnt match regex '%s'"
+                              %(value, expression))
+    return match
+
+# ====================================================
+#  schema for show vlan
+# ====================================================
+class ShowVlanSchema(MetaParser):
+    """Schema for show vlan"""
+    schema = {
+        'vlans':{
+            Any():{
+                Optional('vlan_id'): str,
+                Optional('name'): str,
+                Optional('status'): str,
+                Optional('voice'): str,
+                Optional('jumbo'): str,
```

Comment:
Do all of these need to be optional? Optional keys are best used when it's possible for only part of a line to show up. Overusing optional can lead to parsers passing when they should otherwise have failed